pkg/bindings/containers/logs.go

```diff
@@ -50,7 +50,6 @@ func Logs(ctx context.Context, nameOrID string, opts LogOptions, stdoutChan, std
 	if err != nil {
 		return err
 	}
-	defer response.Body.Close()
```

Comment:
Why was this removed?

Reply:
@mheon It could cause a double close on the body.